tests/datasets/test_spacenet.py

```diff
@@ -14,7 +14,14 @@
 from _pytest.fixtures import SubRequest
```

Comment:
It would be helpful to add one or more `data.py` files that show how the fake test data is generated. We've been adding these for other new datasets and trying to add them to existing datasets. This makes it easy to regenerate the data at a different image size or create the data on the fly during testing (something I would like to do in the future).

Reply:
I was planning to add a `data.py` once all the SpaceNet classes were done.  I can add it when I add `SpaceNet6`.